packages/peregrine/lib/talons/RootComponents/Category/useCategoryContent.js

```diff
@@ -70,8 +91,75 @@ export const useCategoryContent = props => {
     const categoryDescription = categoryData
         ? categoryData.categories.items[0].description
         : null;
+    const sortingMethods = sortData
+        ? sortData?.products?.sort_fields?.options
+        : null;
+    // price needs to be bidirectional
+    const priceSortingPropsDescending = {
+        id: 'sortItem.priceDesc',
+        text: 'Price: High to Low',
+        attribute: 'price',
+        sortDirection: 'DESC'
+    };
+
+    const priceSortingPropsAscending = {
+        id: 'sortItem.priceAsc',
+        text: 'Price: Low to High',
+        attribute: 'price',
+        sortDirection: 'ASC'
+    };
+    const positionSortProps = {
+        id: 'sortItem.position',
+        text: 'Position',
+        attribute: 'position',
+        sortDirection: 'ASC'
+    };
+    const relevanceSortProps = {
+        id: 'sortItem.relevance',
+        text: 'Best Match',
+        attribute: 'relevance',
+        sortDirection: 'DESC'
+    };
+    const defaultSortingOptions = [
+        priceSortingPropsAscending,
+        priceSortingPropsDescending,
+        positionSortProps,
+        relevanceSortProps
+    ];
+    const allSortingMethods = sortingMethods
+        ? sortingMethods
+              .map(method => {
+                  let sortingProps;
+
+                  if (method.value !== 'price' && method.value !== 'position') {
+                      sortingProps = {
+                          ...sortingProps,
+                          id: `sortItem.${method.value}`,
+                          text: `${method.label}`,
+                          attribute: method.value,
+                          sortDirection: 'ASC'
+                      };
+                  }
+
+                  return sortingProps;
+              })
+              .filter(method => method !== undefined)
```

Comment:
Why do we have undefined methods here?

Reply:
`sortingMethods` starts with a `null` value until data is returned from the query. I'm just making sure these values don't go in.

Reply:
But you are escaping falsy values on the beginning at line 130.